src/BookBundle/Resources/views/Front/realisation_search.html.twig

```diff
@@ -18,68 +18,115 @@
+                            {{ form_label(form.Recherche,'Rechercher un projet par son nom ') }}
+                            {{ form_widget(form.Recherche) }}
+                            {{ form_errors(form.Recherche) }}
+                        </div>
+                    </div>
+                    <hr>
+                    <div class="col-lg-5 col-lg-offset-5 projetTransition">
+                        <p>Ou sélectionner un ou plusieurs critères</p>
+                    </div>
+                    <div class="col-lg-12 blocksearch">
                         {{ form_start(form) }}
                         {{ form_errors(form) }}
-                        {{ form_row(form.school) }}
-                        {{ form_row(form.category) }}
-                        {{ form_row(form.promotion) }}
-                        <button type="submit" class="btn btn-default btn-wcs">OK</button>
-                        {{ form_end(form) }}
+                        <div class="row">
+                            <div class="col-lg-2 col-lg-offset-1">
+                                {{ form_label(form.school,'Ecole') }}
+                                {{ form_widget(form.school) }}
+                                {{ form_errors(form.school) }}
+                            </div>
+                            <div class="col-lg-2 col-lg-offset-1">
+                                {{ form_label(form.category,'Categorie') }}
+                                {{ form_widget(form.category) }}
+                                {{ form_errors(form.category) }}
+                            </div>
+                            <div class="col-lg-2 col-lg-offset-1">
+                                {{ form_label(form.promotion,'Promotion') }}
+                                {{ form_widget(form.promotion) }}
+                                {{ form_errors(form.promotion) }}
+                            </div>
+                            <div class="col-lg-2 col-lg-offset-1">
+                                <button type="submit" class="btn btn-default btn-wcs btn-searchProject">OK</button>
+                            </div>
+                        </div>
 
-                        {{ form_start(form) }}
-                        {{ form_row(form.Recherche) }}
                         {{ form_end(form) }}
+
                     </div>
                 </div>
             </div>
+        </div>
 
-            <div class="search-selectors">
-                <div class="row col-xs-offset-2">
-                </div>
+        <!--Projects list -->
+        <div class="list_projects row" id="project-list">
+            <div class="row">
+                {% for projectSearch in projectsSearch %}
+                    <div class="col-xs-6 col-md-5 vignetteProject">
+                        <div class="col-md-12 image-listProjects"
+                                {% for picture in projectSearch.pictures %}{% if picture.isPrincipal == true  %}
```

Comment:
{% for picture in projectSearch.pictures  if picture.isPrincipal == true  %}
en une seule fois et du coup le endif est inutile

je suis tjs pas fan des img en background-image